src/game/Laptop/EMail.cc

```diff
@@ -1538,27 +1538,22 @@ static void DisplayEmailMessageSubjectDateFromLines(Email* pMail, INT32 iViewerY
 	SetFontAttributes(MESSAGE_FONT, FONT_BLACK, NO_SHADOW);
 
 	// all headers, but not info are right justified
-	INT16 usX;
-	INT16 usY;
+	RightAlign const alignment{ MESSAGE_HEADER_WIDTH };
```

Comment:
This is quite confusing even before the change — now all the content is left aligned.

Reply:
I'm not quite sure what you mean? This function only draws the text above the actual email content (the 'From:', 'Day:' and 'Subject:' strings) and those are right aligned with or without this change.

Reply:
I don't see anything right-aligned:
![image](https://github.com/user-attachments/assets/d5c3b120-2264-4563-9368-c18365662308)

Reply:
The strings From:, Day: and Subject: are right aligned. The content of the fields ('Enrico' and '1 are supposed to be left aligned, at least based on the original code. The Subject content ('Inquiry') has this left margin in vanilla too.

It might look better if the Day field was right aligned and the Subject content didn't have that margin but I didn't want touch the vanilla look with this PR and mix code cleanup with a change in behavior.

Reply:
Ahh, I see, thanks.